docs/getting_started.rst

```diff
@@ -0,0 +1,361 @@
+set before. For example, if your prefix is ``!``, you will execute your
+command like this: ``!ping``.
+
+.. note:: Since the prefix can be anything, it'll be referenced as ``[p]``
+    in documentations.
+
+.. _getting-started-commands:
+
+~~~~~~~~~~~~
+The commands
+~~~~~~~~~~~~
+
+The command you're going to use the most is help. That command will
+show you **all of the available commands** of the bot with a small description.
+
+.. code-block:: none
+
+    [p]help
+
+.. tip:: The message is generated dynamically and users will only see the
+    commands they can use. You can edit this using the permissions cog.
+
+You can also pick a command to get its detailled description and the
+parameters.
+
+.. code-block:: none
+
+    [p]help command
+
+The parameters are shown as enclosed in ``< >`` if they're needed, or
+``[ ]`` if optional.
+As an example, the ban command will show this in the help message:
+``Syntax: !ban <user> [days] [reason]``
+
+This means that it is necessary to provide ``user``. However, the
+``days`` value (number of messages to delete) is optional, like
+the ``reason`` value, used for the modlog.
+
+You can use help to show the **categories** too (named cogs).
+Just do so (notice the capitalization):
+
+.. code-block:: none
+
+    [p]help YourCategory
+
+Help also shows **command groups**. They are group of commands.
+To get the description of a subcommand, type this:
+
+.. code-block:: none
+
+    [p]help commandgroup subcommand
+
+When using them, you also need to specify the command group.
+As an example, ``cleanup`` has 6 subcommands. If you want
+to use one, do it like this: ``[p]cleanup messages 10``
+
+.. _getting-started-cogs:
+
+----
+Cogs
+----
+
+Red is built with cogs, fancy term for plugins. They are
+modules that enhance the Red functionnalities. They contain
+commands to use.
+
+Red comes with 19 cogs containing the basic features, such
+as moderation, utility, music, streams...
+
+You can see your loaded and unloaded cogs with the ``[p]cogs``
+command. By default, all cogs will be unloaded.
+
+You can load or unload a cog by using the load or unload command
+
+.. code-block:: none
+
+    [p]load cogname
+    [p]unload cogname
+
+.. tip:: You can load and unload multiple cogs at once:
+
+    .. code-block:: none
+
+        [p]load cog1 cog2 ...
+
+You can enable and disable everything you want, which means you can
+customize Red how you want!
+
+.. _getting-started-community-cogs:
+
+~~~~~~~~~~~~~~
+Community cogs
+~~~~~~~~~~~~~~
+
+There's an entire `community <https://discord.gg/red>`_ that contributes
+to Red. Those contibutors make additional cogs for you to use. You can
+download them using the downloader cog.
+
+You can start using the downloader cog by loading it: ``[p]load downloader``
+
+You can find cogs by searching on `cogs.red <https://cogs.red>`_. Find whatever you want,
+there are hundreds of cogs available!
+
+.. note:: ``cogs.red``, the website that list all of the cogs is not
```

Comment:
I'm not sure if this should even include cogs.red until that is up, no reason to confuse the users tho don't bother reading the note or do not know what bot version they have installed.

Reply:
I was thinking about removing the note as soon as cogs.red is ready, so users would know a better solution is in progress. I can remove the hyperlink to prevent confusion tho.